src/sagemaker/amazon/amazon_estimator.py

```diff
@@ -622,13 +633,25 @@ def get_image_uri(region_name, repo_name, repo_version=1):
         if supported_version:
             return get_xgboost_image_uri(region_name, supported_version[0])
 
-        logging.warning(
-            "There is a more up to date SageMaker XGBoost image. "
-            "To use the newer image, please set 'repo_version'="
-            "'%s'. For example:\n"
-            "\tget_image_uri(region, 'xgboost', '%s').",
-            XGBOOST_VERSION_1,
-            XGBOOST_VERSION_1,
-        )
     repo = "{}:{}".format(repo_name, repo_version)
     return "{}/{}".format(registry(region_name, repo_name), repo)
+
+
+def _xgboost_version_compare(repo_version):
```

Comment:
let's rename this to something that's a little more descriptive about what the return boolean actually means. how about `_is_not_latest_xgboost_version`? (or `_is_latest_xgboost_version` and then using a `not` for the check above?)

Reply:
It's easier to simply check if the version matches the latest version, so I think having this function as `_is_latest_xgboost_version` makes more sense. Then we can check `if not _is_latest_xgboost_version(repo_version)` above.